crates/net/network/src/peers/manager.rs

```diff
@@ -1037,6 +1039,7 @@ pub(crate) enum PeerCommand {
 
 /// Actions the peer manager can trigger.
 #[derive(Debug)]
+#[allow(missing_docs)]
```

Comment:
this allow is not needed i think?

Reply:
can also be resolved by adding a simple one liner to PeerRemoved

Reply:
struct-like variants' fields need docs

Reply:
BTW this whole module/crate is weird, I would appreciate someone taking a look after this PR with `unreachable_pub` because a lot of the public structs reference private ones in docs